src/asset.ts

```diff
@@ -1,119 +1,134 @@
-import axios from 'axios';
 import AinftBase from './ainftBase';
-import { HttpMethod } from './types';
-import { buildData } from './util';
+import { AppCreditInfo, HttpMethod, NftContract, NftToken, UserNfts } from './types';
 
+const prefix = 'asset';
```

Comment:
How about use `this.route` instead of prefix?

Reply:
Oh, it looks better! I fixed to use this.route, thanks!